classes/NewsMatchDonation_Settings.php

```diff
@@ -0,0 +1,221 @@
+<?php
+/**
+ * The class for the News Match Donation plugin's settings
+ */
+
+/**
+ * Register and populate a settings page for the News Match Donation Plugin:
+ *
+ * - org name
+ * - org_id
+ * - live url for donation form
+ * - staging url for donation form
+ * - staging/live toggle
+ * - default salesforce compaign ID
+ * - ???
+ */
+class NewsMatchDonation_Settings {
+	public $settings_page = 'newsmatchdonation';
+	public $settings_group = 'newsmatchdonation_group';
+	public $settings_section = 'newsmatchdonation_section';
+	public $options_prefix = 'nmds_';
+
+	public function __construct() {
+		add_action( 'admin_menu', array( $this, 'register_submenu_page' ) );
+		add_action( 'admin_init', array( $this, 'register_settings' ) );
+	}
+
+	/**
+	 * register the settings page
+	 */
+	public function register_submenu_page() {
+		add_submenu_page(
+			'plugins.php',
+			__( 'News Match Donation Shortcode', 'newsmatch' ),
+			__( 'News Match Shortcode', 'newsmatch' ),
+			'manage_options', //permissions level is this because that seems right for site-wide config options
+			$this->settings_page,
+			array( $this, 'settings_page_output' )
+		);
+	}
+
+	/**
+	 * The settings page output
+	 *
+	 * should output a bunch of HTML
+	 */
+	public function settings_page_output() {
+		if ( ! current_user_can( 'manage_options' ) ) {
+			wp_die( esc_html__( 'You do not have sufficient permissions to access this page.', 'scaip' ) );
+		}
+		?>
+		<div class="wrap newsmatch-admin">
+			<h1><?php esc_html_e( 'News Match Donation Shortcode Options', 'newsmatch' ); ?></h1>
+			<form method="post" action="options.php">
+				<?php
+					settings_fields( $this->settings_group );
+					do_settings_sections( $this->settings_section );
```

Comment:
`do_settings_sections()` takes `$page` as a parameter, but here you're passing the section as the parameter. This is causing your sections to not appear